hastory-server/src/Data/Hastory/Server/HastoryHandler.hs

```diff
@@ -0,0 +1,29 @@
+module Data.Hastory.Server.HastoryHandler
+  ( module Data.Hastory.Server.HastoryHandler
+  , module Control.Monad.Reader
+  ) where
+
+import Control.Monad.Reader
+import Data.Pool
+import Database.Persist.Sql
+import Servant
+import Servant.Auth.Server
+
+type HastoryHandler a = ReaderT ServerSettings Handler a
+
+data ServerSettings =
+  ServerSettings
+    { _ssDbPool :: Pool SqlBackend
+    , _ssJWTSettings :: JWTSettings
+    , _ssCookieSettings :: CookieSettings
```

Comment:
Usually we use `serverSet` as the prefix, and definitely no underscore because we're not using lenses.